.github/workflows/release-qa.yml

```diff
@@ -0,0 +1,41 @@
+# Deploys all the resources to QA environment
+name: QA Release
+on:
+  push:
+    branches:
+      - main
+      - chore/unify-qa-gh-actions
+  workflow_dispatch:  # Supports manual deployment
+
+jobs:
+  bd-deployment:
+    name: Database Deployment
+    uses: ./.github/workflows/db-qa.yml
+    secrets: inherit
+  bd-update:
+    name: Database Update
+    needs: bd-deployment
+    uses: ./.github/workflows/db-update-qa.yml
+    secrets: inherit
+  batch-deployment:
+    name: Batch Processes Deployment
+    needs: bd-update
+    uses: ./.github/workflows/datasets-batch-deployer-qa.yml
+    secrets: inherit
+  api-deployment:
+    name: API Deployment
+    needs: batch-deployment
+    uses: ./.github/workflows/api-qa.yml
+    secrets: inherit
+  integration-tests:
```

Comment:
Should we have ENVIRONMENT defined here?

Reply:
I'll add it; I'm wondering why the release.yml(prod) doesn't have defined the ENVIRONMENT variable.

Reply:
You should check all usages of integration-tests.yml. I think there are other places where it's not defined.

Reply:
I suspect prod should have it also, but it mostly affects the extraction of the commit hash and version. And since we always release when using the latest commit in main, dev and prod coincide at that time.

Reply:
BTW if you look at https://github.com/MobilityData/mobility-feed-api/actions/runs/10741604111 you'll see that there's a `integration-tests-results-dev` in the artefacts, even if it's a prod release. But it only affects the name of the file.

Reply:
Fixed all places with the missing parameter